src/node/markdown/plugins/preWrapper.ts

```diff
@@ -44,5 +44,4 @@ function extractLang(info: string) {
     .replace(/:(no-)?line-numbers({| |$|=\d*).*/, '')
     .replace(/(-vue|{| ).*$/, '')
     .replace(/^vue-html$/, 'template')
-    .replace(/^ansi$/, '')
```

Comment:
This should be kept I guess. It's mainly used for that language at top-right corner and code group titles:

Now:

<img width="712" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/eb7169b1-09b0-49e5-8855-480ca31af501">

Before:

<img width="712" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/30de2501-c4c6-4b62-af4c-a89edd1423b2">